cli/src/extract-tests/index.js

```diff
@@ -250,4 +263,22 @@ async function deleteExtraAddonFiles(analysis) {
     force: true,
     recursive: true,
   });
+
+  await packageJson.modify((json) => {
+    // Remove any test-related scripts, as they now will be in the test-app
+    json.scripts = Object.fromEntries(
+      Object.entries(json.scripts).filter(
+        ([key]) => key !== 'test' && key !== '_test' && !key.match(/^test:.*/)
+      )
```

Comment:
```suggestion
      Object.entries(json.scripts).filter(
        ([key]) => !key.includes('test')
      )
```

Reply:
Yeah, I wanted to do this first, but then thought maybe this is too broad and could include some unwanted changes? But happy to change if you feel comfortable doing so!

Reply:
yeah, I think it's safeish.
The main thing is that it's less logic, and doesn't include the CrowdStrike convention of the leading _.
(I use `_:` in in some other project, for example, so matching on just `test` inclusion would cover all of this)